src/example/com/example/components/datomic.clj

```diff
@@ -21,11 +21,19 @@
                     database "?user=" user "&password=" password)]
     url))
 
+(defn presence
+  "Returns s if s is not blank. Takes an optional default."
+  [s & [default]]
+  (if (str/blank? s)
+    default
+    s))
```

Comment:
You know it's gonna be fun when my first actual code commit to a rapid application framework is coercion